.travis.yml

```diff
@@ -71,7 +71,7 @@ matrix:
       env: CI_BUILD_TARGET="sitltest-quadplane sitltest-plane"
     - if: type != cron
       compiler: "clang-7"
-      env: CI_BUILD_TARGET="sitltest-rover sitltest-sub""
+      env: CI_BUILD_TARGET="sitl sitltest-rover sitltest-sub""
```

Comment:
No, no, no, this already takes way too long and we can't take SITL out of Semaphore without breaking release branches. If it is desired to build more boards with Clang then we should look at supporting it on Semaphore.

Reply:
This is what was decided on the devcall.  It was thought that the incremental time to build all SITL targets here would be minimal.

@WickedShell measured this at an additional 8 minutes (23 -> 31 minutes).

My testing here seems to reflect that 30% premium when *not* using ccache:

```
ccache in play:

with SITL:
real	10m3.126s
user	10m44.149s
sys	2m41.884s

without SITL:
real	9m47.852s
user	10m15.686s
sys	2m36.781s

with SITL (again):
real	10m9.875s
user	10m25.704s
sys	2m33.524s


ccache --clear and rm -rf build

with SITL:

real	19m28.768s
user	10m12.861s
sys	2m25.805s

without SITL:
real	14m34.731s
user	9m43.164s
sys	2m23.088s
```

Why is ccache false in `.travis.yml`?  Weren't we actually preserving our ccache at one stage with CI?

Reply:
We have ccache in CI, but saving it between runs isn't beneficial, after some time it's actually worse to have it - so it has been removed.

It's also not very useful for sitl vs sitltest because sitltest might be running a different version of pymavlink - and that alone might make a large majority of files get recompiled.

In Travis it was 6m20s more compared to latest master. Unfortunately sitltest builds aren't showing the ccache after the build so we don't know hit values, but looking at build times, Rover didn't benefit much (4m20s vs 3m53s), Sub was much faster (4m2s vs 8s).

This is a lot of time and our jobs are already very unbalanced - making them more unbalanced is a bad idea.